client/lib/invites/actions.js

```diff
@@ -91,6 +91,8 @@ export function acceptInvite( invite ) {
 			} else {
 				analytics.tracks.recordEvent( 'calypso_invite_accepted' );
 			}
+
+			callback( error, data );
```

Comment:
I believe we need to make sure that `callback` is defined and is a function here.

![screen shot 2](https://cloud.githubusercontent.com/assets/1126811/12405781/c9fd9ab4-be0c-11e5-9db8-e8e08831c68c.png)